### ISSUE TITLE
`removedb` seems to ignore `--testnet` flag

### ISSUE THREAD

**Issue by konradkonrad** [ethereum/go-ethereum#3200]
#### System information

Geth version: 
```
Geth
Version: 1.4.18-stable-c72f5459
Protocol Versions: [63 62]
Network Id: 1
Go Version: go1.6
OS: linux
GOPATH=/home/konrad/.gopath
GOROOT=/usr/local/go
```
OS & Version: Linux

#### Expected behaviour
I have a datadir at `~/.ethereumdatadirs/mordengeth` with a partially synced testnet chain in `~/.ethereumdatadirs/mordengeth/testnet`.
In order to resync testnet from scratch, I run the command

```
$ geth --testnet --datadir ~/.ethereumdatadirs/mordengeth removedb
Remove local database? [y/N] y
y
Removing chaindata...
Removed in 73.631µs
```

Now starting the client with
```
$ geth --testnet --datadir .ethereumdatadirs/mordengeth
```
should sync from scratch.

#### Actual behaviour
The testnet data was still there, i.e. not syncing from scratch.

#### Steps to reproduce the behaviour
1) Start syncing the testnet in a certain datadir
2) Stop and call `removedb` as described above
3) Check for testnet data in datadir

#### Backtrace
not applicable

**Comment by fjl** [comment]
This was fixed by #2914

**Comment by konradkonrad** [comment]
cool, then that's already covered. thanks!
